contrib/init/sysvinit-redhat/docker

```diff
@@ -97,6 +99,13 @@ rh_status_q() {
     rh_status >/dev/null 2>&1
 }
 
+
+check_for_cleanup() {
+  if [ -f ${pidfile} ]; then
+    /bin/ps -fp $(cat ${pidfile}) > /dev/null || rm ${pidfile}
+  fi
```

Comment:
Well, it solves a valid problem, hacky or not, so I'll leave the reject/accept up to @lsm5 :+1: